src/app_service/service/message/dto/message.nim

```diff
@@ -259,8 +263,13 @@ proc isPersonalMention*(self: MessageDto, publicKey: string): bool =
   return false
 
 proc isGlobalMention*(self: MessageDto): bool =
-  # TODO: we should check here if message contains global mention.
-  return false
+  var containsGlobalMention = false
+  for pText in self.parsedText:
+    for child in pText.children:
+      if (child.literal.contains(MENTION_EVERYONE_LITERAL)):
```

Comment:
Because there can be other types of mentions, "here" mentions, group mentions etc, (and I dunno how many more there'll be in the future) I'd for now hold back on this and let the client decide (we do the same thing for personal mentions btw, so that way it's consistent as well)